src/pages/STG_PanelRD.page

```diff
@@ -1,81 +1,72 @@
 <apex:page controller="STG_PanelRD_CTRL" docType="html-5.0" action="{!init}">
-    <c:STG_PageHeader sectionLabel="{!$Label.stgNavRecurringDonations}" pageLabel="{!$Label.stgNavRecurringDonations}" />
-    <p><apex:outputText value="{!$Label.npe03__RecurringDonationSettingsDescription}" escape="false" /></p>
-
-    <apex:form id="form" styleClass="form-horizontal">
-        <apex:pageMessages id="pageMessages"/>
-        <apex:pageBlock mode="maindetail">
-            <apex:outputPanel layout="block" rendered="{!isReadOnlyMode}" styleClass="button-block">
-                <apex:commandButton value="{!$Label.stgBtnEdit}" status="statusLoad" action="{!editSettings}" rendered="{!isReadOnlyMode}" immediate="true" rerender="form" styleClass="btn-sm btn-primary" />
+    <apex:form id="form" styleClass="slds-m-around--x-large">
+        <c:STG_PageHeader sectionLabel="{!$Label.stgNavRecurringDonations}" pageLabel="{!$Label.stgNavRecurringDonations}" />
+        <p class="slds-text-body--small slds-m-top--x-small">
+            <apex:outputText value="{!$Label.npe03__RecurringDonationSettingsDescription}" escape="false" />
+        </p>
+        <c:UTIL_PageMessages />
+        <div class="slds-grid slds-grid--align-center slds-grid--vertical-align-center slds-p-around--large">
+            <apex:outputPanel layout="block" rendered="{!isReadOnlyMode}" styleClass="slds-button-group">
+                <apex:commandButton value="{!$Label.stgBtnEdit}" status="statusLoad" action="{!editSettings}" rendered="{!isReadOnlyMode}" immediate="true" rerender="form" styleClass="slds-button slds-button--small slds-button--neutral" />
```

Comment:
when you click save or cancel, and the Edit button is redisplayed, it is clipped on its left side.  I reproduced multiple times.  not sure what is specific about this Edit button vs all our other ones!